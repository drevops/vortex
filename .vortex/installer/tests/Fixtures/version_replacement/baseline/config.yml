version: '2.1'

aliases:  
    docker:
      - image: drevops/ci-runner:canary
    docker2:
      - image: drevops/ci-runner:canaryafter
    docker3:
      - image: drevops/ci-runner:beforecanary
    docker4:
      - image: drevops/ci-runner:beforecanaryafter
 
