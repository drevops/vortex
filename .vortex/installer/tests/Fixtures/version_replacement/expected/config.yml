version: '2.1'

aliases:  
    docker:
      - image: drevops/ci-runner:__VERSION__
    docker2:
      - image: drevops/ci-runner:canaryafter
    docker3:
      - image: drevops/ci-runner:beforecanary
    docker4:
      - image: drevops/ci-runner:beforecanaryafter
 
